src/components/instantlaunches/admin/InstantLaunchList.js

```diff
@@ -209,8 +271,10 @@ const InstantLaunchList = ({ showErrorAnnouncer }) => {
         },
     });
 
-    const isLoading = allILs.isLoading || dashboardILs.isLoading;
-    const isError = allILs.isError || dashboardILs.isError;
+    const isLoading =
+        allILs.isLoading || dashboardILs.isLoading || navDrawerILs.isLoading;
```

Comment:
Should we also check for `removeFromDrawerMutation` and `addToNavDrawerMutation` status ?

Reply:
Yes - good catch. There were others missing too.  I'll add those.